app/src/pages/inside/projectSettingsPageContainer/content/emptyStatePage/emptyStatePage.jsx

```diff
@@ -33,16 +33,14 @@ export const EmptyStatePage = ({
   documentationLink,
   title,
   disableButton,
-  integrationEmptyState,
+  customClassName,
```

Comment:
We can put here alot of custom classes for each element maybe descriptionClassName will be more relevant name